src/lib/DALMeteorMethods/GeneralDBMethods.js

```diff
@@ -0,0 +1,23 @@
+/*eslint-disable*/
+Users = new Mongo.Collection.get("users");
+
+if(Meteor.isServer){
+    Meteor.publish("users", function(){
```

Comment:
NO NO NO
Do not publish all user data to client... ever
If you do this, they will have access to every user in our database and all their info. Including login/access tokens, name and email, and any other info that their account (in our case google) stores.

Change this to only publish data we need, maybe userid, name, and picture, nothing else

FYI, if you want to see all the data on a user (most likely yourself on local), just use users.findOne() in browser console... pretty sketch yo